utils/sane-backends/files/saned.init

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh /etc/rc.common
+
+START=90
+STOP=10
+USE_PROCD=1
+
+sanedconf() {
+	local cfg="$1"
+	config_get user "$cfg" user nobody
+	config_get portrange "$cfg" portrange
+	config_get acl "$cfg" acl
+}
+
+start_service() {
+	local user hplip portrange acl
+
+	. /lib/functions.sh
+
+	config_load saned
+	config_foreach sanedconf saned
```

Comment:
I'm no procd expert but foreach seems to be for daemons with repeatable configurations. It does not seem to be your case. AFAIK, your script support only one "config saned".